server_admin/topics/authentication/kerberos.adoc

```diff
@@ -1,239 +1,209 @@
 
 A typical use case for web authentication is the following:
 
-. User logs into his desktop (Such as a Windows machine in Active Directory domain or Linux machine with Kerberos integration enabled).
-. User then uses his browser (IE/Firefox/Chrome) to access a web application secured by {project_name}.
-. Application redirects to {project_name} login.
-. {project_name} renders HTML login screen together with status 401 and HTTP header `WWW-Authenticate: Negotiate`
-. In case that the browser has Kerberos ticket from desktop login, it transfers the desktop sign on information to the {project_name}
-  in header `Authorization: Negotiate 'spnego-token'` . Otherwise it just displays the login screen.
-. {project_name} validates token from the browser and authenticates the user.
-  It provisions user data from LDAP (in case of LDAPFederationProvider with Kerberos authentication support) or let user
-  to update his profile and prefill data (in case of KerberosFederationProvider).
-. {project_name} returns back to the application.
-  Communication between {project_name} and application happens through OpenID Connect or SAML messages.
-  The fact that {project_name} was authenticated through Kerberos is hidden from the application.
-  So {project_name} acts as broker to Kerberos/SPNEGO login.
-
-For setup there are 3 main parts:
-
-. Setup and configuration of Kerberos server (KDC)
-. Setup and configuration of {project_name} server
-. Setup and configuration of client machines
+. The user logs into the desktop.
+. The user accesses a web application secured by {project_name} using a browser.
+. The application redirects to {project_name} login.
+. {project_name} renders HTML login screen with status 401 and HTTP header `WWW-Authenticate: Negotiate`
+. If the browser has a Kerberos ticket from desktop login, the browser transfers the desktop sign-on information to the {project_name} in header `Authorization: Negotiate 'spnego-token'`. Otherwise, it displays the standard login screen, and the user enters their login credentials.
+. {project_name} validates the token from the browser and authenticates the user. {project_name}
+. If using LDAPFederationProvider with Kerberos authentication support, {project_name} provisions user data from LDAP. If using KerberosFederationProvider, {project_name} lets the user update their profile and pre-fill their login data.
+. {project_name} returns to the application. {project_name} and the application communicate through OpenID Connect or SAML messages. {project_name} acts as a broker to Kerberos/SPNEGO login, so {project_name} authenticating through Kerberos is hidden from the application.
+
+Three steps are necessary to set up Kerberos authentication:
```

Comment:
Three steps are necessary to..
Change to "You perform these steps to.."

Reply:
Changed to "Perform the following steps to"